### DISCUSSION TITLE
step

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3153] [Q&A] [answered]
why module 'pybamm' has no attribute 'step'??? @brosaplanella 
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/922b0085-2305-4b19-af02-a3f6fb72ff04)

**Comment by brosaplanella** [comment, marked as answer]
You need to upgrade PyBaMM to the most recent version (23.5).

**Reply by zeecode07** [reply]
How to upgrade?

**Reply by agriyakhetarpal** [reply]
You may upgrade PyBaMM to the latest version via the `pip install --upgrade pybamm` command

**Reply by zeecode07** [reply]
thanks, got it !
